examples/axum-http-service/Cargo.toml

```diff
@@ -9,8 +9,7 @@ edition = "2021"
 tower_otel_http_metrics = { path = "../../", package = "tower-otel-http-metrics", features = ["axum"], default-features = false }
 axum = { features = ["http1", "tokio"], version = "0.7", default-features = false }
 bytes = { version = "1", default-features = false }
-opentelemetry = { version = "0.27", default-features = false }
-opentelemetry_sdk = { version = "0.27", features = ["rt-tokio"], default-features = false }
-opentelemetry-semantic-conventions = { version = "0.27", default-features = false }
-opentelemetry-otlp = { version = "0.27", features = ["metrics", "grpc-tonic"], default-features = false }
+opentelemetry = { version = "0.28", default-features = false }
+opentelemetry_sdk = { version = "0.28", features = ["rt-tokio"], default-features = false }
```

Comment:
"rt-tokio" feature is no longer required.

Reply:
yep, I needed to re-run [unused-features](https://github.com/TimonPost/cargo-unused-features)

Reply:
correction: unused features does not catch this one

Reply:
Hmmh, rt-tokio still seems to be used in some places in `opentelemetry_sdk`, including their own tests. https://github.com/open-telemetry/opentelemetry-rust/blob/main/docs/migration_0.28.md also doesn't mention the feature being deprecated. Maybe this should be cleaned up/clarified in `opentelemetry_sdk` first?

Reply:
In any case, I opened https://github.com/francoposa/tower-otel-http-metrics/pull/22, seems the examples here work fine without it.

Reply:
async_runtime and related are moved to "experimental" feature-flag. They may be removed in the future, but before removing, alternative will be provided.

Reply:
Thanks! Is it possible to add this to the migration guide? It probably makes sense to help people update their feature flags and know about the status.

Reply:
its part of changelog. Prefixing ff with "experimental" is sufficient right? Feel free to send a wording suggestion, if you think the doc/guide is not making it clear.